lms/product/plugin/grouping_service.py

```diff
@@ -62,10 +62,6 @@ def get_groups_for_grading(
 
```

Comment:
I've moved this from the original location, as we now have to register all our plugins. It's also quite often required to refer to them when you are working on implementing child plugins. The idea here is that this directory will list all the plugins available in a way which is more similar to the product specific dirs.

An object like this obviously has affinity with both the implementers / product side and the consumer / service side. In this case there are more ties now on this side.